webapp/src/views/ServiceList.vue

```diff
@@ -24,44 +35,107 @@
           <th scope="col">Vorname</th>
           <th scope="col">Nachname</th>
           <th scope="col">Datum</th>
+          <th scope="col">Kommentar</th>
         </tr>
       </thead>
       <tbody>
         <tr
           v-for="(person, index) in serviceHistory"
           :key="index"
           :class="{
-            requested: person.groupMemberStatus === 'Requested',
-            todelete: person.groupMemberStatus === 'ToDelete',
+            requested: person.groups[0].groupMemberStatus === 'Requested',
```

Comment:
This is a tricky one! I would suggest to remove strike through and have only two visible states: default and inactive (grey).

The logic could be inactive if group member status is not `Active` for at least one group, otherwise default design. For a user to differentiate the states, I would add a pill (inactive, requested) if one these states is consistent across all groups. When states are not consistent, I would add a pill for all groups where member status is not `Active`, e.g. _Kaffeedienst: Inaktiv_, _Kehren am Samstag: zu löschen_. We would leave out the link from group to service at this point but I think that's fine for now.

Reply:
In my opinion the current implementation is sufficient. We can still implement the special case of a single _requested_ or _to delete_ pill at a later time.